charts/helm-dashboard/templates/deployment.yaml

```diff
@@ -46,6 +46,10 @@ spec:
               value: /opt/dashboard/helm/data
             - name: DEBUG
               value: {{- ternary " '1'" "" .Values.debug }}
+            {{- if .Values.image.basePath }}
+            - name: HD_BASE_PATH
+              value: /helm-dashboard
+            {{end}}
```

Comment:
Assuming base path is string, setting `VITE_BASE_PATH` here would be better chart design, not requiring separate image